sana.py

```diff
@@ -0,0 +1,69 @@
+import torch
```

Comment:
is this file meant to be included?

Reply:
Oh, no. This is only for testing. It'll be removed later.